store/src/db/migrations.rs

```diff
@@ -27,6 +28,7 @@ pub static MIGRATIONS: Lazy<Migrations> = Lazy::new(|| {
             PRIMARY KEY (block_num, note_index),
             CONSTRAINT fk_block_num FOREIGN KEY (block_num) REFERENCES block_headers (block_num),
             CONSTRAINT notes_block_number_is_u32 CHECK (block_num >= 0 AND block_num < 4294967296),
+            CONSTRAINT notes_batch_index_is_u32 CHECK (batch_index BETWEEN 0 AND 0xFFFFFFFF)
             CONSTRAINT notes_note_index_is_u32 CHECK (note_index >= 0 AND note_index < 4294967296)
```

Comment:
If we are using `BETWEEN` for `batch_index`, probably makes sense to do the same for `block_num` and `note_index`.

Reply:
@bobbinth yes, I did it, but in different PR: https://github.com/0xPolygonMiden/miden-node/pull/293/files#diff-aaf55c2ae683cbe885dbb5cebb9f8526e16dba91d863f854f4ecca5c366f900aR14-R17